### ISSUE TITLE
non-zero delta for early stopping

### ISSUE THREAD

**Issue by mahdis-saeedi** [fani-lab/OpeNTF#319]
> Hi @hosseinfani 
> 
> In the training phase of the model **imdb.mt10.ts2.e100.ns5.lr0.001.es5.h128.speTrue.lbce.tpw1.tnw0.none.merge_false** with batch size 5, validation loss is too low and train loss is exactly 0 from the first epoch, and this consistently continue until the epoch 99. this is also repeated for all folds.
>  
> <img width="1351" height="616" alt="Image" src="https://github.com/user-attachments/assets/af3f034e-8f4f-412a-b7d3-be215b65b27e" />
> 
> If i correctly understand from your explanation in our last meeting, in this setting it happens because we don't have any negative samples  and the loss is computed only for positive samples which are a few. So this leads to dropping the loss in this case.
> 
> Now if we can allocate a small positive number to delta in the following line, then the model will be early stopped without continuing until the last epoch. 
> 
> https://github.com/fani-lab/OpeNTF/blob/e91816388e0f73b57407bac96ca63481b5bf52fd/src/mdl/earlystopping.py#L31
>  

 _Originally posted by @mahdis-saeedi in [#305](https://github.com/fani-lab/OpeNTF/issues/305#issuecomment-3263400512)_

**Comment by mahdis-saeedi** [comment]
Hi @hosseinfani ,
The code runs without any issues after the change you have merged #321.  I think we can close this issue. Thank you.
